### PR TITLE
Added blog post about vertx-junit5-web-client

### PR DESCRIPTION
**Pull request by slinkydeveloper**
Signed-off-by: francesco <francescoguard@gmail.com>

### CODE REVIEW

File: src/site/blog/posts/2019-07-26-vertx-junit5-web-client.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        // Assert that response contains a particular header
+        responseHeader("x-my", "bar")
+      )
+      .send(testContext); // Complete (or fail) the VertxTestContext
+  }
+
+}
+```
+
+`testRequest()` will use Vert.x Web Client to send the request. When the response is received, It succeds the test or It correctly propagates assertion failures, if any.
```
Comment (jponge) [inline src/site/blog/posts/2019-07-26-vertx-junit5-web-client.md]: There as `It` that should be `it` 😉

**Comment by jponge** [comment]
@slinkydeveloper Why haven't we merged that one?

**Comment by slinkydeveloper** [comment]
Don't know honestly, we can merge it now :)

**Comment by jponge** [comment]
Ok I’ll let you do your checks and merge :-)

**Pull request merged**